kolibri/core/kolibri_plugin.py

```diff
@@ -38,6 +38,7 @@ class KolibriCoreFrontEnd(KolibriFrontEndPluginBase):
     """
     entry_file = "assets/src/kolibri_core_app.js"
     external = True
+    core = True
```

Comment:
Why is this added to `class KolibriCoreFrontEnd(KolibriFrontEndPluginBase)` but not `class KolibriCore(KolibriPluginBase)`?